src/main/java/Ui.java

```diff
@@ -0,0 +1,34 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Ui {
+    public static final String HI_MESSAGE = "Hello! I'm BotChat.\n What can I do for you?";
+    public static final String BYE_MESSAGE = "Bye. Hope to see you again soon!";
+
+    public static String hiMessage() {
+        return HI_MESSAGE;
+    }
+
+    public static String byeMessage() {
+        return BYE_MESSAGE;
+    }
```

Comment:
Would it be better to rename the methods as verbs, i.e. `printHiMessage()` and `printByeMessage()` respectively?

Reply:
Thank you for your review. I have edited my code to enhance readability and adhere to the coding standards as you have suggested.